deploy/secrets-store-csi-driver-windows.yaml

```diff
@@ -48,13 +48,15 @@ spec:
               cpu: 10m
               memory: 20Mi
         - name: secrets-store
-          image: us.gcr.io/k8s-artifacts-prod/csi-secrets-store/driver:v0.0.14
+          image: us.gcr.io/k8s-artifacts-prod/csi-secrets-store/driver:v0.0.15
           args:
             - "--debug=true"
             - "--endpoint=$(CSI_ENDPOINT)"
             - "--nodeid=$(KUBE_NODE_NAME)"
             - "--provider-volume=C:\\k\\secrets-store-csi-providers"
-            - "--metrics-addr=:8080"
+            - "--metrics-addr=:8095"
+            - "--enable-secret-rotation=false"
```

Comment:
setting the flags in the deploy manifest with the defaults so users can easily change it